sbt/src/sbt-test/project/unified/test

```diff
@@ -0,0 +1,13 @@
+> root/Compile/compile
+
+# "Global" now works in shell. optional whitespace arond /
+> show Global / cancelable
+
+# "ThisBuild" now works in shell. optional whitespace arond /
+> show ThisBuild / scalaVersion
+
+> show Test / test
```

Comment:
How do we know that we're calling the right `Test / test`? This tests that the parser accepts it and it runs something, but we don't know which something has been run.